templates/server/src/main/resources/archetype-resources/api/pom.xml

```diff
@@ -29,11 +29,24 @@
       <groupId>com.devonfw.java.modules</groupId>
       <artifactId>devon4j-security</artifactId>
     </dependency>
-    <!-- Required for secrutiy REST service -->
+    <!-- Required for security REST service -->
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <!-- Tests -->
+    <dependency>
+      <groupId>com.devonfw.java.modules</groupId>
+      <artifactId>devon4j-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <!-- EqualsVerifier -->
+    <dependency>
+      <groupId>nl.jqno.equalsverifier</groupId>
+      <artifactId>equalsverifier</artifactId>
+      <version>3.1.9</version>
```

Comment:
As we are proposing this as the default/standard for devonfw projects, we should even better add this dependency to `devon4j-test` directly so all projects get it from there implicitly.

Reply:
Thanks for the comment. I'll add the dependency to the test module.